### PR TITLE
Tbh 218 js box ussd tbcheck

### PR DESCRIPTION
**Pull request by Buhle79**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# NDOH MomConnect Apps
+w wwwwqeq# NDOH MomConnect Apps
```
Comment (erikh360) [inline README.md]: This doesn't look right

File: go-app-ussd_tb_check.js [hunk 2]
```diff
@@ -834,11 +836,7 @@ go.app = function () {
           function (response) {
             answers.group_arm = response.data.tbconnect_group_arm.toLowerCase();
             answers.tbcheck_id = response.data.id;
-            /*self.im.user.answers = {
```
Comment (erikh360) [inline go-app-ussd_tb_check.js:837]: We can remove commented out code

File: go-app-ussd_tb_check.js [hunk 3]
```diff
@@ -876,9 +874,8 @@ go.app = function () {
     self.states.add("state_display_arm_message", function (name) {
       var answers = self.im.user.answers;
       var arm = answers.group_arm;
-      console.log("####", arm);
```
Comment (erikh360) [inline go-app-ussd_tb_check.js:879]: We can also remove all debug console.logs

File: go-app-ussd_tb_check.js [hunk 4]
```diff
@@ -955,40 +952,84 @@ go.app = function () {
...
+        choices: [new Choice("state_get_nearest_clinic", $("Next"))],
       });
     });
 
-    self.add("state_clinic_list", function (name) {
+    self.add("state_get_nearest_clinic", function (name, opts) {
+      var answers = self.im.user.answers;
+      var lng = answers.location_lng;
+      var lat = answers.location_lat;
+      console.log(typeof lat, typeof lng);
```
Comment (bearnard) [inline go-app-ussd_tb_check.js]: delete

File: go-app-ussd_tb_check.js [hunk 5]
```diff
@@ -955,40 +952,84 @@ go.app = function () {
...
+//      self.im.config.healthcheck.url + "/v1/clinic_finder?longitude="+ lng +"&latitude="+ lat);
```
Comment (bearnard) [inline go-app-ussd_tb_check.js]: delete

File: go-app-ussd_tb_check.js [hunk 6]
```diff
@@ -955,40 +952,84 @@ go.app = function () {
...
+
+      return new JsonApi(self.im)
+        /*.get(self.im.config.healthcheck.url + "/v1/clinic_finder?longitude="+ lng +"&latitude="+ lat,*/
```
Comment (bearnard) [inline go-app-ussd_tb_check.js]: del

File: go-app-ussd_tb_check.js (hunk 5 above)
Comment (erikh360) [inline go-app-ussd_tb_check.js]: More commented out code

**Review by erikh360 — COMMENTED** [review]
I think do a little bit of cleanup before I have a look again.

We don't want any commented out code or debug console.logs

File: package.json [hunk 7]
```diff
@@ -31,7 +31,7 @@
   },
   "devDependencies": {
     "csv-parse": "^4.14.0",
-    "grunt": "^1.4.1",
+    "grunt": "^1.5.3",
```
Comment (bearnard) [inline package.json]: why the upgrade?

File: src/ussd_tb_check.js [hunk 8]
```diff
@@ -484,6 +484,8 @@ go.app = function () {
               self.pad_location(location.lat, 2) +
               self.pad_location(location.lng, 3) +
               "/";
+            self.im.user.location_lat = location.lat;
```
Comment (bearnard) [inline src/ussd_tb_check.js]: I don't see where `self.im.user.location_lat` is used anywhere after you set it here?

**Comment by bearnard** [comment]
👍

**Pull request merged**